### PR TITLE
fix: add necessary peerDependencies

### PR DESCRIPTION
**Pull request by ylemkimon**
**What kind of change does this PR introduce?**
Bugfix

**Did you add tests for your changes?**
N/A

**If relevant, did you update the documentation?**
N/A

**Summary**
This PR adds `webpack` and `webpack-cli` peerDependencies to packages requiring `@webpack-cli/utils`, which has: https://github.com/webpack/webpack-cli/blob/134c77a8e52ad17a73a8e61c0ce3ca8277ef2ddf/packages/utils/package.json#L23-L25

Dependency tree (D: dependency, P: peer dependency, *: added in this PR, **: reason):
```
webpack-cli
├─D> @webpack-cli/package-utils
├─D> @webpack-cli/info
│    └─P> webpack-cli
├─D> @webpack-cli/init
│    ├─P> webpack*
│    ├─P> webpack-cli*
│    ├─D> @webpack-cli/generators
│    │    ├─P> webpack*
│    │    ├─P> webpack-cli*,**
│    │    ├─D> @webpack-cli/package-utils
│    │    ├─D> @webpack-cli/utils
│    │    │    ├─P> webpack**
│    │    │    ├─P> webpack-cli**
│    │    │    └─D> @webpack-cli/package-utils
│    │    └─D> @webpack-cli/webpack-scaffold
│    └─D> @webpack-cli/utils
│         ├─P> webpack**
│         ├─P> webpack-cli**
│         └─D> @webpack-cli/package-utils
└─D> @webpack-cli/serve
     └─P> webpack-cli
```

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by ylemkimon** [comment]
I wonder why the non-existent version of `webpack-cli@5` is listed as peerDependencies for `@webpack-cli/utils`: https://github.com/webpack/webpack-cli/blob/134c77a8e52ad17a73a8e61c0ce3ca8277ef2ddf/packages/utils/package.json#L25

File: packages/generators/package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "yeoman-generator": "4.7.2"
   },
   "peerDependencies": {
-    "webpack-cli": "3.x.x || 4.x.x"
+    "webpack": "4.x.x || 5.x.x",
+    "webpack-cli": "3.x.x || 4.x.x || 5.x.x"
```
Comment (evenstensberg) [inline packages/generators/package.json:24]: we don't have a v5 for cli

Comment (ylemkimon) [reply packages/generators/package.json:24]: @evenstensberg `webpack-cli@5` is listed as peerDependencies of `@webpack-cli/utils`: https://github.com/webpack/webpack-cli/blob/134c77a8e52ad17a73a8e61c0ce3ca8277ef2ddf/packages/utils/package.json#L25

I'll remove it from this PR. Should I remove it from the above as well?

Comment (evilebottnawi) [reply packages/generators/package.json:24]: Yes

Comment (ylemkimon) [reply packages/generators/package.json]: Done

**Review by evilebottnawi — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Comment by rishabh3112** [comment]
Thanks

**Pull request merged**